internal/printer/emitcontext.go

```diff
@@ -917,3 +927,37 @@ func (c *EmitContext) VisitIterationBody(body *ast.Statement, visitor *ast.NodeV
 
 	return updated
 }
+
+func (c *EmitContext) SetSyntheticLeadingComments(node *ast.Node, comments []SynthesizedComment) *ast.Node {
```

Comment:
Not this PR, but there are some callers of this I left commented out in the node builder (probably more than that though, just opining)

Reply:
Sent #1607 before this task slipped out of my brain.